tests/test_variable_manipulation.py

```diff
@@ -275,3 +276,102 @@ def test_find_all_variables(df_vartypes):
 
     with pytest.raises(KeyError):
         assert _find_all_variables(df_vartypes, non_existing_vars)
+
+
+def test_find_categorical_and_numeric_variables_one_categorical_variables(
```

Comment:
could we put all these tests together (into 1) and use parametrize? 

also, can we use a df that already exists? like df_vartypes?

Reply:
It seems that the conftest dataframes are either (1) not recognized if they are not imported or (2) stay as a function object.